### PR TITLE
Add `Clear Software Updates settings` Saga

### PR DESCRIPTION
**Pull request by jamie-suse**
# Description

Adds a Redux Saga to allow the user to clear _Software Updates_ settings.

## How was this tested?

Added tests for the Saga.

### CODE REVIEW

File: assets/js/state/sagas/softwareUpdatesSettings.js [hunk 1]
```diff
@@ -61,6 +63,19 @@ export function* updateSoftwareUpdatesSettings(payload) {
...
+export function* clearSoftwareUpdatesSettings() {
+  yield put(startLoadingSoftwareUpdatesSettings());
+
+  try {
+    yield call(clearSettings);
+
+    yield put(setEmptySoftwareUpdatesSettings());
+  } catch (error) {
+    const errors = get(error, ['response', 'data', 'errors'], []);
+    yield put(setSoftwareUpdatesSettingsErrors(errors));
```
Comment (dottorblaster) [inline assets/js/state/sagas/softwareUpdatesSettings.js:75]: I think there's no need to set errors, this is meant for validation errors in other sagas

**Review by dottorblaster — CHANGES_REQUESTED** [review]
Just a tiny change request

**Review by nelsonkopliku — APPROVED** [review]
Besides what Alessio shared, LGTM

**Review by dottorblaster — APPROVED** [review]
go

**Pull request merged**